docs/manifest.json

```diff
@@ -31,6 +31,10 @@
             {
               "title": "Instrumenting a K8s Cluster",
               "path": "/docs/guides/user/instrumenting-a-k8s-cluster.md"
+            },
+            {
+              "title": "Collecting Cloud Metrics",
```

Comment:
The title is kind of ambiguous in this context.  I would add something else...   like "Collecting Cloud Vendor Metrics" or "Collecting Native Cloud Vendor Metrics".

Reply:
Went with "Collecting Cloud Vendor Metrics" since its around the same length as the other titles in the index. I could see "Collecting Native Cloud Metrics" working too